plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/BannedElementTest.java

```diff
@@ -43,6 +43,7 @@ public void testBannedElementConstructor_nullLocation() {
         null);
   }
   
+  @SuppressWarnings("unlikely-arg-type")
```

Comment:
Hmm, maybe we should settle on some common settings because I don't see any warnings here, and I've checked that I have all _unlikely_ error/warnings on.

Reply:
![image](https://user-images.githubusercontent.com/1005544/33841112-c5fd1064-de64-11e7-989e-490df9670016.png)

This is actually a very good warning. In almost all cases it would catch a real bug. Here it doesn't only because we are specifically testing equals returning false when comparing a different object type.